sse2neon.h

```diff
@@ -8482,12 +8482,12 @@ const static uint8_t _sse2neon_cmpestr_mask8b[16] ALIGN_STRUCT(16) = {
 
 #define PCMPSTR_EQ_16x8(a, b, mtx) \
```

Comment:
You can rename `EIGHT` to `x8` and `SIXTEEN` to `x16`. Later, make use of catenate trick. See https://github.com/sysprog21/rv32emu/blob/master/src/common.h